js/client.js

```diff
@@ -869,6 +869,24 @@
 			}
 
 			switch (parts[0]) {
+			case 'authconfig':
```

Comment:
Is `authconfig` the right name for this?

Maybe just `groups` or something. We still officially call them groups, don't we?

`customauth`?